src/Illuminate/Http/ApiResponseTrait.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Illuminate\Http;
+
+use InvalidArgumentException;
+use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Pagination\AbstractPaginator;
```

Comment:
Then we should also require `"illuminate/pagination": "5.5.*"` in `Illuminate/Http/composer.json`